src/amp-story-player/amp-story-player-impl.js

```diff
@@ -630,6 +649,28 @@ export class AmpStoryPlayer {
     this.isLaidOut_ = true;
   }
 
+  /**
+   * Fetches more stories from the publisher's endpoint.
+   * @return {!Promise}
+   * @private
+   */
+  fetchStories_() {
+    const url = this.readPlayerConfig_().behavior.endpoint;
+    const init = {
+      method: 'GET',
+      headers: {
+        Accept: 'application/json',
+      },
+    };
+
+    return fetch(url, init)
```

Comment:
I really think we should at the very least add an `offset` parameter, otherwise it's almost impossible to build an infinite carousel

Reply:
Looking for other "infinite scroll" implementations in AMP, I found how amp-next-page does it. It looks for a `next` attribute on the response, and if specified, sets it as the next endpoint to which we should send the request to get the next stories.

I like this approach because it's easy to know when to stop fetching for more -- when there is no "next" object specified in the request. I made the changes in the latest commit, lmk what you think.